about-page.html

```diff
@@ -0,0 +1,153 @@
+          <li>
+            <a href="#contact-me" class="menu-list nav-styled" id="elem3">Partners</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <!-- popup menu area -->
+
+    <header class="header-wrapper">
+        
+      <div class="menu-icon hide2" id="open"><i class="fa fa-reorder"></i></div>
+    </header>
+    <main>
+      <section class="about-section1">
+        <section class="bottom-section">
+          <h3 class="greetings">"Hello!smart world"</h3>
+          <h1 class="top-header">ITU EVENT</h1>
+        </section>
+        <section class="text-container text-containera">
+          <p class="text1 text1a">
+            This Forum was collocated with the ITU-T Study Group 20: Internet of
+            things (IoT) and smart cities and communities (SC&C) meeting which
+            took place from 3-13 December and the Joint Coordination Activity on
+            Internet of Things and Smart Cities and Communities (JCA-IoT and
+            SC&C) meeting which took place on 6 December 2018 in the same venue.
+          </p>
+        </section>
+        <section class="text2 contact-text-resize text2b">
+          <p>
+            Please contact us per Email for any further question about the ItU
+            Event, global forum on Artificial Intelligence and Internet of
+            things<br /><b><u>ituGlobalForum@itu.ca</u></b>
+          </p>
+        </section>
+      </section>
+      <section class="middle1">
+        <p class="logo-header">
+          The Global Forum of Artificial Intelligence and Iot logo
+        </p>
+      </section>
+      <div class="bar-cover"><div class="bar"></div></div>
+      <div class="logo-description logo-descriptiona">
+        <p>
+          The logo of the Global Forum of Artificial Intelligence and Iot was
+          decided through the logo competition of may 2016.
+        </p>
+      </div>
+      <div class="logo-wrapper logo-wrappera">
+        <img class="logo" src="./IMG/itu-logo.png" alt="logo" />
+      </div>
+      <div class="hrline"></div>
+      <section class="middle1">
+        <p class="logo-header">
+          See the past of the Artificial Intelligence and Iot Forum
+        </p>
+      </section>
+      <div class="bar-cover"><div class="bar"></div></div>
+      <div class="logo-description">
+        <p>
+          Take a look at the last two AI and Iot Global Forums which took place
+          in New York and in London.
+        </p>
+      </div>
+      <div class="events">
+        <div class="new-york">
+          <div class="inside">
+            <h2 class="date">2016</h2>
+            <p>AI and Iot Forum in New-York</p>
+          </div>
+        </div>
+        <div class="london">
+          <div class="inside">
+            <h2 class="date">2017</h2>
+            <p>AI and Iot Forum in London</p>
+          </div>
+        </div>
+      </div>
+      <div class="partners partnersa">
+        <h2 class="logo-header font">Partners</h2>
```

Comment:
- Kindly uses semantic tags, instead of divs use more of `section` for best practices.